Algorithms/Graphs/Detect_Cycle_In_Undirected_Graph.cpp

```diff
@@ -0,0 +1,71 @@
+// C++ program to detect cycle in an undirected graph using BFS. 
+#include <bits/stdc++.h> 
+using namespace std; 
+
+void addEdge(vector<int> adj[], int u, int v) 
+{ 
+	adj[u].push_back(v); 
+	adj[v].push_back(u); 
+} 
+
+bool isCyclicConntected(vector<int> adj[], int s, 
```

Comment:
What is s, V?

Reply:
Changed the notation of s, and commented what V means